src/status_im/ui/screens/accounts/recover/styles.cljs

```diff
@@ -1,14 +1,16 @@
 (ns status-im.ui.screens.accounts.recover.styles
   (:require-macros [status-im.utils.styles :refer [defstyle]])
-  (:require [status-im.ui.components.colors :as colors]))
+  (:require [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.react :as react]))
+
+(def window-width (:width (react/get-dimensions "window")))
+(def window-height (:height (react/get-dimensions "window")))
+(defn scaled-x [n] (* (/ window-width 375) n))
```

Comment:
Where do we get those numbers from?

Reply:
Screen dimensions in DP of the phone in Figma designs (iPhone X)